package.json

```diff
@@ -39,6 +38,7 @@
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
     "lint:docs": "markdownlint \"**/*.md\"",
     "lint:eslint-docs": "npm-run-all \"update:eslint-docs -- --check\"",
+    "lint:fix": "yarn lint:js --fix && yarn lint:package-json-sorting:fix",
```

Comment:
Can you open a separate PR for this? If we're going to add it, I'd like to do it a bit differently, so better for a separate PR.

Reply:
can do.

How do you fix anything without this tho? :sweat_smile:

Reply:
https://github.com/ember-cli/eslint-plugin-ember/pull/2036